doc/source/icepack_index.rst

```diff
@@ -411,7 +411,13 @@ either Celsius or Kelvin units).
    "shortwave", ":math:`\bullet` flag for shortwave parameterization ('default' or 'dEdd')", ""
    "sil", "silicate concentration", "mmol/m\ :math:`^3`"
    "sk_l", "skeletal layer thickness", "0.03 m"
-   "snoice", "snow-ice formation", "m"
+   "snowage_drdt0", "snowage table 3D data for drdt0 (10^-6 m/hr)", ""
+   "snowage_kappa", "snowage table 3D data for kappa (10^-6 m)", ""
+   "snowage_rhos", "snowage table dimension data for rhos (kg/m^3)", ""
+   "snowage_T", "snowage table dimension data for temperature (deg K)", ""
+   "snowage_tau", "snowage table 3D data for tau (10^-6 m)", ""
+   "snowage_Tgrd", "snowage table dimension data for temp gradient (deg K/m)", ""
+   "snoice"", "snow-ice formation", "m""
```

Comment:
too many "" in this line, after snoice and after m

Reply:
I can fix these things.  Do you want me to update the PR or do you want to fix them as you move forward?

Reply:
Please go ahead and fix them.  Thx!